README.md

```diff
@@ -188,6 +189,29 @@ $ngReduxProvider.createStoreWith(reducers, [thunk, 'myInjectableMiddleware']);
 
 Middlewares passed as **string** will then be resolved throught angular's injector.
 
+## Config
+
+### Debouncing the digest
+You can debounce the digest triggered by store modification (usefull in huge apps with a  lot of store modifications) by passing a config parameter to the `ngReduxProvider`.
+
+```javascript
+import angular from 'angular';
+
+angular.module('ngapplication').config(($ngReduxProvider) => {
+  'ngInject';
+
+  // eslint-disable-next-line
+  $ngReduxProvider.config.debounce = {
+    wait: 100,
+    mawWait: 500,
```

Comment:
Looks like there's a typo here, I assume this is intended to be `maxWait`

Reply:
You're right, thx for review.
It's corrected.